src/edu/ucsd/library/xdre/web/CollectionOperationController.java

```diff
@@ -1971,4 +1985,29 @@ private static void appendArksFromFileInput(List<String> items, String fileInput
 			}
 		}
 	}
+
+	/*
+	 * Test conversion errors
+	 * @param rdfXML
+	 * @throws UnsupportedEncodingException
+	 * @throws IOException
+	 * @throws TransformerException
+	 */
+	private static void testConversion(Document rdfDoc) throws UnsupportedEncodingException, IOException, TransformerException, DocumentException {
+		File tmpFile = File.createTempFile("bib", "-rdf.xml");
```

Comment:
would this ever get run in parallel (multi-threaded) and risk having a name collision? I'm not familiar with the details of `createTempFile`, so perhaps that's handled already?

Reply:
Good question! I haven't tested the `createTempFile` method in the multiple threaded environment. But from the documentation I think the filename should be unique for any successful calls:
```
 If this method returns successfully then it is guaranteed that:

    The file denoted by the returned abstract pathname did not exist before this method was invoked, and
    Neither this method nor any of its variants will return the same abstract pathname again in the current invocation of the virtual machine. 
```
I think it won't be bad to put it in a synchronized block since it should be executed fast enough that won't affect the performance. What do you think?